frontend/components/ui/blocknote/blocks/latex-block.tsx

```
@@ -0,0 +1,229 @@
+          setHasError(false);
+          return;
+        }
+
+        try {
+          const html = katex.renderToString(content, {
+            throwOnError: false,
+            displayMode: true, // Always use display mode
+          });
+          setPreviewHtml(html);
+          setHasError(false);
+        } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)